domain/model/state/modelstate.go

```diff
@@ -147,6 +149,13 @@ FROM model
 		return coremodel.ReadOnlyModel{}, errors.Trace(err)
 	}
 
+	if credOwnerName != "" {
```

Comment:
It might be worth adding an info log to state that the owner is empty.

Reply:
Added, had to pipe a logger in